src/Components/Authorization/commonElements.tsx

```diff
@@ -28,31 +24,40 @@ export const GrayFacebookButton = FacebookButton.extend.attrs({
   margin-top: 0;
   font-size: 12px;
 
+  div {
+    margin: 0px;
+  }
+
+  span {
+    margin-left: 5px;
+  }
+
   &:hover:not(:disabled) {
     background: #fff;
   }
 `
 
-export const BlockButton = props => (
-  <InvertedButton block>{props.children}</InvertedButton>
-)
-
-export const TOSCheckbox = ({ error, errorMessage, value, ...props }) => (
-  <Checkbox {...{ error, errorMessage, checked: value }}>
+export const TOSCheckbox = ({ error, name, onChange, value, ...props }) => (
+  <StyledCheckbox {...{ checked: value, error, onChange, name }}>
```

Comment:
This was surprisingly tricky -- couldn't figure out why checkboxes weren't getting errors and it was because `handleChange` uses the `name` property to register changes.